A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js

```diff
@@ -701,30 +701,16 @@ var Airbus_FMA;
             return false;
         }
         IsActive_LVRCLB() {
-            if (Airbus_FMA.CurrentPlaneState.radioAltitude > 1.5) {
-                if (Airbus_FMA.CurrentPlaneState.flightPhase == FlightPhase.FLIGHT_PHASE_TAKEOFF) {
-                    if (Airbus_FMA.CurrentPlaneState.highestThrottleDetent < ThrottleMode.CLIMB && Simplane.getIndicatedSpeed() > 30) {
-                        return true;
-                    }
-                }
-                else if (Airbus_FMA.CurrentPlaneState.flightPhase == FlightPhase.FLIGHT_PHASE_CLIMB) {
-                    if (Airbus_FMA.CurrentPlaneState.highestThrottleDetent > ThrottleMode.CLIMB) {
-                        if (Airbus_FMA.CurrentPlaneState.radioAltitude >= Simplane.getThrustReductionAltitude()) {
-                            return true;
-                        }
-                    }
-                }
-                else if (Airbus_FMA.CurrentPlaneState.flightPhase == FlightPhase.FLIGHT_PHASE_GOAROUND) {
-                    if (Airbus_FMA.CurrentPlaneState.highestThrottleDetent > ThrottleMode.CLIMB) {
-                        if (Airbus_FMA.CurrentPlaneState.radioAltitude >= Simplane.getThrustReductionAltitude()) {
-                            return true;
-                        }
-                    }
-                    else if (Airbus_FMA.CurrentPlaneState.highestThrottleDetent < ThrottleMode.CLIMB) {
-                        return true;
-                    }
+            const thrustReductionAltitude = Simplane.getThrustReductionAltitude();
+
+            if (Airbus_FMA.CurrentPlaneState.radioAltitude > thrustReductionAltitude) {
+                if (Airbus_FMA.CurrentPlaneState.flightPhase === FlightPhase.FLIGHT_PHASE_CLIMB) {
```

Comment:
Would this not break if you have the acceleration altitude wasn't reached first?

Acceleration altitude changes the flight phase from 'TAKEOFF' to 'CLIMB', so if you hit the THR RED (thrust reduction altitude) before the ACC (acceleration altitude) I don't think it would return true in your logic here.

Let me know if you Discord if you want, maybe I'm just misreading it! 👍